meal-prep-frontend/src/components/Week.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Day } from './Day';
+
+const daysOfTheWeek = [
+    'Monday',
+    'Tuesday',
+    'Wednesday',
+    'Thursday',
+    'Friday',
+    'Saturday',
+    'Sunday'
+]
+
+export const Week = () => {
+    return (
+    <div>
```

Comment:
Would having each day semantically described with the `<section>` tag be more descriptive than a div?